proc_dash/utility.py

```diff
@@ -65,13 +64,21 @@ def extract_pipelines(bagel: pd.DataFrame) -> dict:
     return pipelines_dict
 
 
+def get_id_columns(data: pd.DataFrame) -> list:
+    """Returns names of columns which identify a given participant record"""
+    return (
```

Comment:
minor, but: is there a reason to do this over just `if ... return; else return`? I find that cleaner to read

Reply:
Ah, mainly brevity. In some past PRs I've gotten suggestions to switch simple if-else (that is, returning a single object) blocks to ternary operators, so I guess the preference depends on who's reviewing the code 😅. I would like to keep the style consistent across the app, if possible. Would you recommend defaulting to if ... return; else return ?

Reply:
I see, I think this is still readable as is. I would not use ternary if this got any longer. Consistency is a good idea, so if you've used this elsewhere, then let's keep it